vitest.config.ts

```diff
@@ -26,8 +33,18 @@ export default mergeConfig(
         },
       }),
     ],
+    resolve: {
+      alias: {
+        '#app': resolve(__dirname, './node_modules/nuxt/dist/app'),
+      },
+    },
     test: {
       environment: 'happy-dom',
+      env: parsed,
+      globals: true,
+      sequence: {
+        shuffle: false, // change back to true later
```

Comment:
We'll want to toggle this to `true` sometime in the future. This way we make sure that tests are passing only on their own setup, and not because of side effects from previous tests.

Reply:
Created an issue to track this:
https://github.com/leaderboardsgg/leaderboard-site/issues/534